host-configs/CVX/cvx-tpls.cmake

```diff
@@ -0,0 +1,6 @@
+# Pavel: edit that to provide the path for thirdPartyLibs
+#        alternatively (and preferably) use
+#          python scripts/config-build.py -hc host-configs/cvx-wsl.cmake -bt Release -D GEOSX_TPL_DIR=/full/path/to/thirdPartyLibs
+#set(GEOSX_TPL_DIR "/home/ptls/thirdPartyLibs/install-cvx-wsl-debug" CACHE PATH "" FORCE)
+
+include(${CMAKE_CURRENT_LIST_DIR}/../tpls.cmake)

```

Comment:
including a file which merely only include another file may too complicated.
Could you directly include the `tpls.cmake` instead of the current file?

Reply:
i am just not sure where to place that comment which is very useful for beginners like me
i would need to duplicate it then

Reply:
You can define it using `-DGEOSX_TPL_DIR=...` in the command line?

Reply:
Somehow it doesn't work all the time from IDEs, so hard codded values can be also useful.